### PR TITLE
WT-10566 Fix uninitialized variables

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: src/reconcile/rec_col.c [hunk 1]
```diff
@@ -869,6 +869,13 @@ __wt_rec_col_fix(
             upd = NULL;
             /* Make sure not to apply an uninitialized time window, or one from another key. */
             WT_TIME_WINDOW_INIT(&unpack.tw);
+
+            /* Initialize the selected update. */
+            upd_select.upd = NULL;
```
Comment (etienneptl) [inline src/reconcile/rec_col.c:874]: Should we have a macro to init those too? Given we already have one for the time window, we could have a `clear` macro that inits everything and calls `WT_TIME_WINDOW_INIT`.

Comment (quchenhao) [reply src/reconcile/rec_col.c]: done

**Review by etienneptl — APPROVED** [review]

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by Sean04 — APPROVED** [review]
LGTM

**Comment by quchenhao** [comment]
evergreen merge

**Pull request merged**